### PR TITLE
bug(web): make mobile tooltip handling explicit

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
The tooltips aren't working well in the mobile dev tools.

## Description
This PR updates the tooltip interactions to be more mobile specific and display a more consistent behavior across platforms

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
I wonder if this will fix the interaction with my Apple Pen as well 👀

**Comment by VIKTORVAV99** [comment]
> > I wonder if this will fix the interaction with my Apple Pen as well 👀
> 
> 
> 
> Would love for you to test 🖊️ 📱 

Not sure if that's possible via the emulator but I'll definitely give it a go if I can!

**Comment by tonypls** [comment]
> > > I wonder if this will fix the interaction with my Apple Pen as well 👀
> > 
> > 
> > Would love for you to test 🖊️ 📱
> 
> Not sure if that's possible via the emulator but I'll definitely give it a go if I can!

Would it work to visit the branch preview on iPad?

**Comment by VIKTORVAV99** [comment]
> > > > I wonder if this will fix the interaction with my Apple Pen as well 👀
> > > 
> > > 
> > > Would love for you to test 🖊️ 📱
> > 
> > 
> > Not sure if that's possible via the emulator but I'll definitely give it a go if I can!
> 
> Would it work to visit the branch preview on iPad?

Just tried it and I'm having the same issue as before (not sure if it's the same you are trying to fix though).

Basically all hover stuff work with the pen but not clicks. So I can't press on things with the pen but I can hover on things. This applies both to the Map tooltip and the Chart tooltips.

Normal buttons and interactive elements work fine.

**Comment by cadeban** [comment]
> Would it work to visit the branch preview on iPad?

I doubt it will work for iPad, since useIsMobile checks if the breakpoint is 'sm' which most iPads are larger than:
https://github.com/electricitymaps/electricitymaps-contrib/blob/9451ea2bf431715782c9ab0c2320024681f84123/web/src/utils/styling.ts#L37

Works so much better on smaller screens though!

**Comment by VIKTORVAV99** [comment]
> > Would it work to visit the branch preview on iPad?
> 
> I doubt it will work for iPad, since useIsMobile checks if the breakpoint is 'sm' which most iPads are larger than:
> 
> https://github.com/electricitymaps/electricitymaps-contrib/blob/9451ea2bf431715782c9ab0c2320024681f84123/web/src/utils/styling.ts#L37
> 
> Works so much better on smaller screens though!

Yeah it's using the "desktop" version of the app.

File: web/src/components/tooltips/TooltipWrapper.tsx [hunk 1]
```diff
@@ -19,10 +19,9 @@ export default function TooltipWrapper({
   side,
   sideOffset,
   tooltipClassName,
-  isMobile,
```
Comment (VIKTORVAV99) [inline web/src/components/tooltips/TooltipWrapper.tsx:22]: Nice to get rid of this prop drilling.

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Pull request merged**